lib/menu.ts

```diff
@@ -99,11 +99,11 @@ function baseElem(createFn: MenuClassCons, triggerElem: Element, createFunc: Men
  *    --weaseljs-selected-color
  *    --weaseljs-menu-item-padding
  */
-export function menuItem(action: () => void, ...args: DomElementArg[]): Element {
+export function menuItem(action: (item: Element) => void, ...args: DomElementArg[]): Element {
```

Comment:
Could probably make it `HTMLElement` (I often find that more helpful), at least if it doesn't require casts anywhere.